saleor/order/emails.py

```diff
@@ -110,7 +109,7 @@ def send_fulfillment_confirmation(order_pk, fulfillment_pk):
 
 
 def send_fulfillment_confirmation_to_customer(order, fulfillment, user):
-    send_fulfillment_confirmation.delay(order.pk, fulfillment.pk)
+    send_fulfillment_confirmation(order.pk, fulfillment.pk)
```

Comment:
Emails should be a task. This is not solving the root of the issue.

Reply:
> "As I said, that's not really ready to merge solution, more like reporting issue with fast proposal if someone is using automatic fulfillment."

Reply:
Created issue for that instead, and added mention for this PR. Closing this PR as it's not needed